### ISSUE TITLE
TypeError: Can't convert 'PosixPath' object to str implicitly

### ISSUE THREAD

**Issue by AlJohri** [AlJohri/docx2pdf#1]
Issue submitted via email by Gary Benson:

```
command: docx2pdf test.docx
  0%|                                                                                                     | 0/1 [00:00<?, ?it/s]Traceback (most recent call last):
  File "docx2pdf", line 8, in <module>
    sys.exit(cli())
  File "docx2pdf/__init__.py", line 170, in cli
    convert(args.input, args.output, args.keep_active)
  File "docx2pdf/__init__.py", line 104, in convert
    return macos(paths, keep_active)
  File "docx2pdf/__init__.py", line 63, in macos
    for line in run(cmd):
  File "docx2pdf/__init__.py", line 54, in run
    process = subprocess.Popen(cmd, stderr=subprocess.PIPE)
  File "python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "python3.5/subprocess.py", line 1490, in _execute_child
    restore_signals, start_new_session, preexec_fn)
TypeError: Can't convert 'PosixPath' object to str implicitly
```

**Comment by fermumen** [comment]
I have this same issue right now.

**Comment by AlJohri** [comment]
@fermumen are you on the latest version? can you post your exact error?

**Comment by fermumen** [comment]
I'm using docx2pdf v0.1.5 on python 3
i'm running 
`convert("./data/docx/file.docx")` on a jupyter notebook.
This is the error I get:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-63-43dcb5afed33> in <module>()
      1 from docx2pdf import convert
      2 
----> 3 convert("./data-raw/pgi/docx/B061-CTP0000.docx")

/Users/Fer/anaconda/lib/python3.5/site-packages/docx2pdf/__init__.py in convert(input_path, output_path, keep_active)
    102     paths = resolve_paths(input_path, output_path)
    103     if sys.platform == "darwin":
--> 104         return macos(paths, keep_active)
    105     elif sys.platform == "win32":
    106         return windows(paths, keep_active)

/Users/Fer/anaconda/lib/python3.5/site-packages/docx2pdf/__init__.py in macos(paths, keep_active)
     61     total = len(list(Path(paths["input"]).glob("*.docx"))) if paths["batch"] else 1
     62     pbar = tqdm(total=total)
---> 63     for line in run(cmd):
     64         try:
     65             msg = json.loads(line)

/Users/Fer/anaconda/lib/python3.5/site-packages/docx2pdf/__init__.py in run(cmd)
     52 
     53     def run(cmd):
---> 54         process = subprocess.Popen(cmd, stderr=subprocess.PIPE)
     55         while True:
     56             line = process.stderr.readline().rstrip()

/Users/Fer/anaconda/lib/python3.5/subprocess.py in __init__(self, args, bufsize, executable, stdin, stdout, stderr, preexec_fn, close_fds, shell, cwd, env, universal_newlines, startupinfo, creationflags, restore_signals, start_new_session, pass_fds)
    945                                 c2pread, c2pwrite,
    946                                 errread, errwrite,
--> 947                                 restore_signals, start_new_session)
    948         except:
    949             # Cleanup if the child failed starting.

/Users/Fer/anaconda/lib/python3.5/subprocess.py in _execute_child(self, args, executable, preexec_fn, close_fds, pass_fds, cwd, env, startupinfo, creationflags, shell, p2cread, p2cwrite, c2pread, c2pwrite, errread, errwrite, restore_signals, start_new_session)
   1488                             errread, errwrite,
   1489                             errpipe_read, errpipe_write,
-> 1490                             restore_signals, start_new_session, preexec_fn)
   1491                     self._child_created = True
   1492                 finally:

TypeError: Can't convert 'PosixPath' object to str implicitly
```

**Comment by AlJohri** [comment]
@fermumen I found the bug. fixed here: https://github.com/AlJohri/docx2pdf/commit/2c1a74066ba8b5bdbf9cc01b32dd3e37da61bde6

published 0.1.6. can you try it?

**Comment by AlJohri** [comment]
For reference, I missed this because this feature exists in Python 3.6+ on MacOS and Python 3.8+ on Windows:

>Changed in version 3.6: args parameter accepts a path-like object if shell is False and a sequence containing path-like objects on POSIX.
>Changed in version 3.8: args parameter accepts a path-like object if shell is False and a sequence containing bytes and path-like objects on Windows.
Source: https://docs.python.org/3/library/subprocess.html#subprocess.Popen

**Comment by fermumen** [comment]
Thank you for the fix, It works perfectly now (tested on 0.1.7 rather than 0.1.6)
